tests/integration/test_data/test_nitrate/NitrateExport.test_existing_dryrun.yaml

```diff
@@ -6,7 +6,7 @@ nitrate.xmlrpc_driver:
   single_request_with_cookies:
   - metadata:
       guess_type: Tuple
-      latency: 0.772697925567627
+      latency: 0.8307182788848877
```

Comment:
@lukaszachy, @jscotka, I see there's a bunch of changes which do not seem to be related to the new `--release` feature. Are they really needed? Could/should we ignore this? Or at least use the `requre-patch` cleanup using `make requre` to prevent repeated changes of the `latency` & friends?

Reply:
Applied `requre-patch purge` in 497679fbec1c2cf97b2ff466a4bf306892eef0e9.